### PR TITLE
Snaps interface

### PR DESCRIPTION
**Pull request by shishirgoenka**
#### What does this PR do?
This PR removes the requirement of performing -s after the -b option in snaps-boot
Also this PR creates separate interface file for the static cfg in /etc/network/interfaces.d/ folder
#### Do you have any concerns with this PR?
No
#### How can the reviewer verify this PR?
By deploying the -b option and validating if the static ip is also reflected
#### Any background context you want to provide?
NA
#### Screenshots or logs (if appropriate)
NA
#### Questions:
- Have you connected this PR to the issue it resolves?
PR #121 
- Does the documentation need an update?
Yes
- Does this add new Python dependencies?
No
- Have you added unit or functional tests for this PR?
No
- Does this patch update any configuration files?
No

### CODE REVIEW

**Comment by RandyLevensalor** [comment]
@shishirgoenka please confirm that I didn't mess anything up when I corrected the merge conflict.

**Comment by shishirgoenka** [comment]
@RandyLevensalor there were issues in the merging done, as the code was removed. I have updated the pxe_utils.py file and not it seems to be fine.

File: snaps_boot/provision/hardware/pxe_utils.py [hunk 1]
```diff
@@ -735,6 +739,32 @@ def __static_ip_configure(static_dict, proxy_dict):
...
+            for part in line.split():
+                if "rootpw" in part:
+                    next_word = list_word[list_word.index("rootpw") + 1]
+
+    user_name = 'root'
+    password = next_word
+    check_dir = os.path.isdir(consts.SSH_PATH)
+    if not check_dir:
+        os.makedirs(consts.SSH_PATH)
+    for i in range(len(host)):
```
Comment (spisarski) [inline snaps_boot/provision/hardware/pxe_utils.py]: please change for loop to:

for this_host in host:
    target = this_host.get('access_ip')

File: snaps_boot/provision/hardware/pxe_utils.py [hunk 2]
```diff
@@ -735,6 +739,32 @@ def __static_ip_configure(static_dict, proxy_dict):
...
+        target = host[i].get('access_ip')
+        iplist.append(target)
+    for i in range(len(host)):
```
Comment (spisarski) [inline snaps_boot/provision/hardware/pxe_utils.py]: Please see the same as above on the for loop

**Comment by shishirgoenka** [comment]
we have updated and removed the conflict in accordance to the snaps-boot master. Request to kindly review and merge.

**Comment by spisarski** [comment]
If you would like to debug on this virtual system, the floating IP of the build machine is 10.197.113.132 with the credentials "ubuntu/password". The IPs of the nodes are 10.1.0.11, 12, & 13 and can be accessed via the build server via "sudo ssh 10.1.0.1x"

**Comment by shishirgoenka** [comment]
@RandyLevensalor , the "import subprocess" was removed during the merging on the PR. this has been updated and now shall be working fine. 
Request you to please test the setup with the changes committed.

**Pull request closed without merging**